app/src/main/java/com/douglasbruce/tasky/features/task/navigation/TaskNavigation.kt

```diff
@@ -0,0 +1,25 @@
+package com.douglasbruce.tasky.features.task.navigation
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavOptions
+import androidx.navigation.compose.composable
+import com.douglasbruce.tasky.features.task.TaskRoute
+
+const val taskNavigationRoute = "task"
+
+fun NavController.navigateToTask(navOptions: NavOptions? = null) {
+    this.navigate(taskNavigationRoute, navOptions)
+}
```

Comment:
If you take the time to create a separate nav function just to get to the task screen, it might be worth to make using it even more convenient by passing the required parameters directly and delegating the creation of `NavOptions` to this `navigateToTask` function